setup.py

```diff
@@ -270,10 +270,10 @@ def EXTENSION(modname):
         include_package_data = True,
         ext_modules = extensions,
         cmdclass = cmdclass,
-        classifiers = ['Development Status :: 3 - Alpha',
+        classifiers = ['Development Status :: 5 - Production/Stable',
                        'Intended Audience :: Science/Research',
                        'Operating System :: Linux, macOS',
-                       'License :: OSI Approved :: MIT License',
+                       'License :: GPLv3',
```

Comment:
Seems to me that GPLv3 is also an OSI approved license :https://opensource.org/licenses/alphabetical. 
So we could still have "OSI Approved" mentioned there?

Reply:
Agreed, have just put it back in.